internal/testing/testdata/testdata.go

```diff
@@ -982,9 +982,7 @@ var (
 			  "CurrentPackage":{
 				 "name":"foreign-types-shared",
 				 "namespace":"",
-				 "qualifiers":[
-					
-				 ],
+				 "qualifiers":null,
```

Comment:
do not merge until we resolve issue with duplicates from deps.dev

Reply:
Ready to review! No more duplicates for `hasSourceAt` and `CertifyVuln`